docs/source/author/topics.rst

```diff
@@ -41,19 +41,16 @@ diagram.
 
 This is just a broad overview of the topics application.
 
-
-.. _viewing-the-current-state:
-
-Viewing the Current State
+Viewing All Topics Content
 ------------------------------------------------------------------------------
 
-When developing locally, once you have a server and ``gulp`` running (see
-:doc:`../getting_started/basic_usage`) you can go to the url below to get a
+When developing locally, once you have run ``csu start`` (see
```

Comment:
I was unsure where to strike the balance of putting that in, because it really comes down to the user understanding their operating system. I only put it once because I know it would work in that situation as the user is in the correct working directory. From that point onwards, I can't be sure that they are in the right place. Should we just assume that and always put `./` before? Or should we assume understanding of how to run a file? Thoughts?